### ISSUE TITLE
Enhancement: Use XDG_DOWNLOAD_DIR if possible instead of hardcoded $HOME/Downloads

### ISSUE THREAD

**Issue by padlyuck** [webinstall/webi-installers#542]
For now, installers use a hardcoded path for downloading packages
```
WEBI_PKG_PATH="${HOME}/Downloads/webi/${PKG_NAME:-error}/${WEBI_VERSION:-latest}"
```
It would be great if installers would try to get XDG_DOWNLOAD_DIR value instead of creating `~/Downloads` near to existing `~/downloads`

**Comment by coolaj86** [comment]
Interesting...

What system uses `~/downloads` instead of `~/Downloads`?

The problem with `XDG_*` is that it never really took off (or at least it hasn't worked where I've tried it), and it's only supported on some Linuxes - not Mac or Windows.

I suppose we could do:

```sh
WEBI_DOWNLOAD_DIR="${XDG_DOWNLOAD_DIR:-${HOME}/Downloads}"
WEBI_PKG_PATH="${WEBI_DOWNLOAD_DIR}/webi/${PKG_NAME:-error}/${WEBI_VERSION:-latest}"

```

Would you care to make that PR?

**Comment by padlyuck** [comment]
@coolaj86 done #547

**Comment by coolaj86** [comment]
Closed via #547
